cdap-ui/app/cdap/services/WizardStores/AddNamespace/AddNamespaceStore.js

```diff
@@ -60,6 +61,21 @@ const defaultPreferencesState = Object.assign({
   }
 }, skippableDefaultState);
 
+const defaultEditableFieldsState = Object.assign({
+  fields: [
+    'name',
```

Comment:
Can this be,
```
Object.keys({
  ...defaultGeneralState,
  ...defaultSecurityState,
  ...defaultPreferencesState
})
```
I am afraid tomorrow we might add a field to one of the other objects and forget to add here. Removes the necessity to hard code the same property in two places.

Reply:
Also for simplicity can the editable fields be an array? do we need `editableFields.fields`?

Reply:
We still want it to be an object, since we want to pass the `__complete: true` property to it. This lets the `Wizard` component know that the step is complete, so the 'Finish' button can be enabled in the wizard. This is done in the `checkRequiredSteps()` method in the `Wizard` component.